File: gaia-sdk-java/gaia-sdk-graphql/src/main/kotlin/gaia/sdk/request/type/ConnectKnowledge.kt
package gaia.sdk.request.type

import gaia.sdk.client.Type
import gaia.sdk.request.intf.*
import gaia.sdk.client.Input
import gaia.sdk.Uuid
import gaia.sdk.ISO8601
import gaia.sdk.Struct
import gaia.sdk.request.input.*
import gaia.sdk.request.enumeration.*

class ConnectKnowledge: Type() {

    fun node(nodeId : Uuid, config: ConnectNodeKnowledge.() -> Unit) = add {
        val name1 = it.register("nodeId", nodeId)
        "node(nodeId:$name1){" + ConnectNodeKnowledge().apply(config).render(it) + "}"
    }
}

File: gaia-sdk-javascript/src/graphql/response/type/ConnectKnowledge.ts

import {ConnectNodeKnowledge} from "./ConnectNodeKnowledge";

import {Uuid, ISO8601, Struct} from "../../GaiaClient";
import {Order} from "../../request/enumeration/Order";
import {OrderByField} from "../../request/enumeration/OrderByField";
import {EdgeOrderByField} from "../../request/enumeration/EdgeOrderByField";

export interface ConnectKnowledge {
    node?:ConnectNodeKnowledge
}